### PR TITLE
kubevirt: ssp: Add CI for branch release-v0.21

### PR DESCRIPTION
**Pull request by akrejcir**

### CODE REVIEW

File: ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.21.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
+base_images:
+  metrics-linter:
+    name: prom-metrics-linter
+    namespace: ci
+    tag: v0.0.2
```
Comment (0xFelix) [inline ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.21.yaml:5]: Was this ever updated sometime?

**Review by 0xFelix — COMMENTED** [review]
/lgtm

File: ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.21.yaml (hunk 1 above)
Comment (akrejcir) [reply ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.21.yaml:5]: Looks like it was: https://quay.io/repository/kubevirt/prom-metrics-linter?tab=tags&tag=latest

But it is out of the scope of this PR, and probably our team. I can open an issue.

Comment (0xFelix) [reply ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.21.yaml:5]: Please do, thanks!

**Comment by akrejcir** [comment]
/pj-rehearse pull-ci-kubevirt-ssp-operator-release-v0.21-e2e-upgrade-functests

File: ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.21.yaml (hunk 1 above)
Comment (akrejcir) [reply ci-operator/config/kubevirt/ssp-operator/kubevirt-ssp-operator-release-v0.21.yaml:5]: Issues are disabled on this repository.

@assafad , you were the author of the PR that added this linter. Do you think it should be updated?

**Pull request merged**